package.json

```diff
@@ -27,11 +32,14 @@
     "identity-obj-proxy": "^3.0.0",
     "if-env": "^1.0.4",
     "jest": "^21.2.1",
-    "preact-cli": "^2.2.1",
+    "lerna": "^3.13.1",
+    "node-sass": "^4.5.3",
+    "preact-cli": "^2.0.2",
+    "preact-cli-plugin-typescript": "^0.2.2",
     "preact-render-to-string": "^3.7.0",
     "preact-widget-scripts": "^0.6.0",
+    "sass-loader": "^6.0.6",
```

Comment:
I don't think we need the `sass-loader` nor the `node-sass`, can we get rid of this?